pkgs/desktops/gnome/extensions/gjs-osk/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  jq,
+}:
+
+let
+  uuid = "gjsosk@vishram1123.com";
+in
+stdenv.mkDerivation rec {
+  pname = "gjs-osk";
+  version = "unstable-2024-09-24";
```

Comment:
Oh, and also the `version` should be prefixed with last version number https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

Reply:
It doesn't show the version on its GitHub repo. Should I use the version on https://extensions.gnome.org?

Currently I have used 0